front-end/src/app/shared/components/fec-intl-tel-input/fec-intl-tel-input.component.ts

```diff
@@ -0,0 +1,77 @@
+import { AfterViewInit, Component, ElementRef, OnDestroy, Optional, Self, ViewChild } from '@angular/core';
+import { ControlValueAccessor, NgControl } from '@angular/forms';
+import * as intlTelInput from 'intl-tel-input';
+
+@Component({
+  selector: 'app-fec-intl-tel-input',
+  templateUrl: './fec-intl-tel-input.component.html',
+  styleUrls: ['./fec-intl-tel-input.component.scss'],
+})
+export class FecIntlTelInputComponent implements AfterViewInit, OnDestroy, 
+  ControlValueAccessor {
+  @ViewChild('telInput') telInput: ElementRef<HTMLInputElement> | null = null;
+
+  private iti: intlTelInput.Plugin | null = null;
```

Comment:
I see that this comes from the example code from their site, but could we spell this out more.  When i first encountered `iti` in the spec above I couldn't even guess what it meant.